src/app/app.component.html

```diff
@@ -56,13 +56,15 @@
 	</div>
 
 </nav>
-
+<!-- <wim-sidebar *ngIf="router.url.indexOf('/settings') === -1 && router.url.indexOf('/profile') === -1 && router.url.indexOf('/gagestats') === -1"></wim-sidebar> -->
+<gs-sidebar *ngIf="router.url.indexOf('/gagestats') > -1"></gs-sidebar>
```

Comment:
To keep consistent with style changes, we should maybe move this to after line 63. That'll help with styling.